### DISCUSSION TITLE
Files are marked as empty in PRs when the only thing that changed is permissions

### DISCUSSION THREAD

**Discussion by hwittenborn** [community/community#57137] [Pull Requests]
### Select Topic Area

Bug

### Body

In a PR, a file is marked as being empty when the only thing that changed on the file is the permissions of it.

This repository/PR demonstrates the issue:

Repo: https://github.com/hwittenborn/pr-permission-demo
PR: https://github.com/hwittenborn/pr-permission-demo/pull/1

**Comment by hutson** [comment]
Similar to #6814
